context_builder.go

```diff
@@ -119,6 +129,10 @@ func (cb *ContextBuilder) Build() (ContextWithHeaders, context.CancelFunc) {
 		timeout = defaultTimeout
 	}
 
+	if cb.TracingDisabled {
+		cb.span.EnableTracing(false)
```

Comment:
Yeah looks weird, but it's a single function to set true/false. It's not really worth breaking the public API for it, especially not as part of this change.